sotodlib/preprocess/processes.py

```diff
@@ -518,12 +518,21 @@ class SubtractHWPSS(_Preprocess):
     """
     name = "subtract_hwpss"
 
+    def __init__(self, step_cfgs):
+        self.hwpss_stats = step_cfgs.get('hwpss_stats', 'hwpss_stats')
+
+        super().__init__(step_cfgs)
+
     def process(self, aman, proc_aman):
-        hwp.subtract_hwpss(
-            aman,
-            hwpss_template = aman[self.process_cfgs["hwpss_extract"]],
-            subtract_name = self.process_cfgs["subtract_name"]
-        )
+        if not(proc_aman[self.hwpss_stats] is None):
+            modes = [int(m[1:]) for m in proc_aman[self.hwpss_stats].modes.vals[::2]]
```

Comment:
I had to add this because `hwpss_model` was used before which is wrapped into `aman` during the `calc` step of  `EstimateHWPSS` however when running `load_preprocess_tod` this fails because that relies only on what's in `proc_aman`/what's in the `process` step.